src/redux/note-details/reducers.ts

```diff
@@ -87,7 +87,7 @@ export const NoteDetailsReducer: Reducer<NoteDetails, NoteDetailsAction> = (stat
   }
 }
 
-const TASK_REGEX = /(\s*[-*] )(\[[ xX]])( .*)/
+const TASK_REGEX = /(\s*(?:[-*+]|\d+[.)]) )(\[[ xX]])( .*)/
```

Comment:
Also technically any symbol other than ` ` (space) should yield a checked checkbox.

Reply:
See https://github.github.com/gfm/#task-list-items-extension-

Reply:
I don't know whether we want this behaviour. I fear it leds to many false-positive recognized strings.

If we want to change it, we'd need to change our markdown-it plugin as well.

Reply:
I have no strong feelings with this. I just noticed this while scrolling through the spec…

Reply:
As long as all the ordering characters work I'm fine with it. 🤷